lib/bobs/scala/step.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# IH_CORE_DIR will be set to the directory containing the bin and lib directories.
+
+function ih::setup::bobs.scala::help() {
+  echo 'This will install the correct scala
```

Comment:
Kinda curious whether this is needed. Isn't bazel supposed to handle all scala-related dependencies itself?

Reply:
Yeah, this is for the IDE itself, which wants Scala stuff I guess. That's also why Go needs to be installed.